### PR TITLE
QFJ-22 update documentation for Acceptor configurations

### PR DESCRIPTION
**Pull request by arthuroz**
including failover, SocketAcceptPort and SocketAcceptAddress

### CODE REVIEW

**Comment by chrjohn** [comment]
Thanks for the PR. Don't be confused because of the failing LGTM JavaScript check. Unfortunately, it is not possible to turn this off.

**Comment by arthuroz** [comment]
> Thanks for the PR. Don't be confused because of the failing LGTM JavaScript check. Unfortunately, it is not possible to turn this off.

Hi Chris,
Yea, could be related to a bug in LGTM regarding `[2020-05-26 10:25:18] [build] No JavaScript or TypeScript code found.` 
I wonder whether I should point the PR to release_1_1, what do you think @chrjohn ?

**Comment by chrjohn** [comment]
The next release will be 2.2.0 so no need to merge to 2.1 branch.

File: quickfixj-core/src/main/doc/usermanual/usage/acceptor_failover.html [hunk 1]
```diff
@@ -9,10 +9,10 @@ <h1>QuickFIX/J User Manual</h1>
 </div>
 		<h2>Simple Failover for Socket-based Acceptors.</h2>
 		<p>
-		When using a MessageStore that supports shared data (FileStore, JdbcStore,
-			SleepycatStore) and that implements the new <code>RefreshableMessageStore</code> interface,
+		When using a MessageStore that supports shared data (FileStore, JdbcStore and SleepycatStore),
 			the Session can be configured to refresh the store information upon
-			logon. You would typically run two acceptor processes using a
+			logon with configuration <span style="color:grey">RefreshMessageStoreAtLogon=Y</span>.
```
Comment (chrjohn) [inline quickfixj-core/src/main/doc/usermanual/usage/acceptor_failover.html:14]: Actually the configuration for this is `RefreshOnLogon`.
https://github.com/quickfix-j/quickfixj/blob/a8725ad9a849dbfcb4c8f220538daa721029c101/quickfixj-core/src/main/java/quickfix/Session.java#L299

Comment (arthuroz) [reply quickfixj-core/src/main/doc/usermanual/usage/acceptor_failover.html]: Good catch, updated.

**Pull request merged**